### PR TITLE
AMP wip

### PR DESCRIPTION
**Pull request by marineb**

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -38,6 +38,9 @@ public function registerBundles()
             new Knp\Bundle\TimeBundle\KnpTimeBundle(),
 
             new AppBundle\AppBundle(),
+
+            // AMP
+            new Takeit\Bundle\AmpHtmlBundle\TakeitAmpHtmlBundle(),
```
Comment (tgalopin) [inline app/AppKernel.php]: Can you put this before the AppBundle?

File: app/Resources/amp/amp-theme/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        },
+        "publisher": {
+          "@type": "Organization",
+            "logo": { "@type": "ImageObject", "url": "https://assets-cdn.github.com/images/modules/logos_page/GitHub-Mark.png", "width": 125, "height": 60 },
+          "name": "Takeit"
+        }
+      }
+    </script>
+    <style amp-boilerplate>
+        {% include '@amp_theme/css/amp-boilerplate.html.twig' %}
```
Comment (tgalopin) [inline app/Resources/amp/amp-theme/index.html.twig]: You should use `{{ include(...) }}` instead.

File: app/Resources/amp/amp-theme/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    </style>
+      {% include '@amp_theme/css/amp-boilerplate-noscript.html.twig' %}
```
Comment (tgalopin) [inline app/Resources/amp/amp-theme/index.html.twig]: Same here

File: app/Resources/amp/amp-theme/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    <style amp-custom>
+        {% include '@amp_theme/css/amp-custom.html.twig' %}
```
Comment (tgalopin) [inline app/Resources/amp/amp-theme/index.html.twig]: Same here

File: app/Resources/amp/amp-theme/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    <title>{{ object.title }}</title>
+    <link rel="canonical" href="{{ url('blog_post', { slug: object.slug }) }}">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <script type="application/ld+json">
+      {
+        "@context": "http://schema.org",
+        "@type": "NewsArticle",
+        "mainEntityOfPage": "{{ url('blog_post', { slug: object.slug }) }}",
+        "headline": "{{ object.title }}",
+          "datePublished": "{{ object.publishedAt|date('Y-m-d H:i:s') }}",
```
Comment (tgalopin) [inline app/Resources/amp/amp-theme/index.html.twig]: Weird indentation

File: app/Resources/amp/amp-theme/index.html.twig [hunk 6]
```diff
@@ -0,0 +1,60 @@
+{#
+Available variables:
+- object - Object of an class instance which you provided in the bundle config. e.g. AppBundle\Entity\Post
+#}
+<!doctype html>
+<html ⚡ lang="en">
+  <head>
+    <meta charset="utf-8">
```
Comment (tgalopin) [inline app/Resources/amp/amp-theme/index.html.twig]: Globally, could you use 4 spaces indentation?

File: app/Resources/amp/amp-theme/README.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+# Demo AMP HTML Theme
```
Comment (tgalopin) [inline app/Resources/amp/amp-theme/README.md]: I guess we can remove this file

**Comment by HeahDude** [comment]
@tgalopin may we close here now that https://github.com/EnMarche/en-marche.fr/pull/1291 is merged?

**Pull request closed without merging**